docs/README.md

```diff
@@ -0,0 +1,65 @@
+<img src="assets/Forge_logo.svg">
+
+MinecraftForge [![Latest Release](https://img.shields.io/maven-metadata/v/https/files.minecraftforge.net/maven/net/minecraftforge/forge/maven-metadata.xml.svg?label=latest)](https://files.minecraftforge.net) [![Discord](https://img.shields.io/discord/313125603924639766.svg?color=%237289da&label=discord&logo=discord&logoColor=%237289da)](https://discord.gg/UvedJ9m) [![Support](https://img.shields.io/badge/patreon-support-orange.svg)](https://www.patreon.com/LexManos) [![Merch](https://img.shields.io/badge/flocker-merch-blueviolet.svg)](https://forgedev.flocker.tv/)
+=============
+
+Forge is a free, open-source modding API all of your favourite mods use!
+
+| Version  | Support |
+| ------------- | ------------- |
+| 1.14.2  | Maintained  |
+| 1.12.2  | Legacy  |
```

Comment:
1.12.2 will not be in legacy support until 1.14 has a recommended build.

Reply:
Oh whoops, I got ahead of myself. 1.13.2 is no longer being maintained though is it?

Reply:
Also, should I keep "Maintained"? Maybe "Active (latest)" and "Active (stable)"?